introduction.md

```diff
@@ -1 +1,10 @@
 Hi, I'm Matt Young. My favorite path in Codeschool is JavaScript so far.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?